index.js

```diff
@@ -143,29 +186,36 @@ function nowServing(line) {
   return `Currently serving ${line.shift()}.`
 }
 
+ticket = 1
```

Comment:
Be careful of creating variables without `let` or `const` in front. Doing that creates a special kind of global variable that can have some unexpected behavior in our code.